shared/build.gradle.kts

```diff
@@ -89,6 +91,14 @@ android {
 		minSdk = 26
 	}
 
+	composeOptions {
+		kotlinCompilerExtensionVersion = "1.5.7"
```

Comment:
The same line persist in `androidApp/build.gradle.kts` I didn't extract it intentionally. I would like to check how renovate bot will take care such case 😄